templates/import.html.twig

```diff
@@ -24,7 +24,8 @@
 <form>
     {{ include('components/pager.html.twig', {
         count: total_count,
-        href: '#',
+        href: get_plugin_web_dir('jamf') ~ '/front/import.php',
```

Comment:
![image](https://github.com/pluginsGLPI/jamf/assets/7335054/ca1d50d5-82f1-4112-b83a-67e37f593707)

I may be wrong, but shouldn't there be a ```?``` instead of ```&```

Reply:
seems good with ```?```

like ```events_list.html.twig```

```twig
<div class="search-footer card-footer">
                  {{ include('components/pager.html.twig', {
                     'href': target ~ '?',
                     'additional_params': 'sort=' ~ sort ~ '&order=' ~ order,
                     'count': count
                  }) }}
               </div>
```

Reply:
If you apply the GLPI patch first or test of latest 10.0/bugfixes, it should be using `?` instead of `&` and code that calls the pager template shouldn't be manually appending `?` or `&` to the href anymore.

Reply:
Sorry, I didn't see the reference to the other PR.